### ISSUE TITLE
Matplotlib 1.5.3 doesn't work in Bash On Windows

### ISSUE THREAD

**Issue by cvax** [conda/conda#4095]
With Anaconda2 v4.2.0, the packaged version of Matplotlib is 1.5.3-np111py27_0. It does not work when used in Bash On Windows because it is bundled with Qt 5 and for some reason it wants Qt 4. If you tried to just downgrade PyQt it still doesn't work. Solution I found was to downgrade to matplotlib 1.5.1.

```
sudo apt-get install libqtgui4
conda install matplotlib=1.5.1
```
Only with this and exporting DISPLAY to VcXsrv does matplotlib work. Is there any way to get 1.5.3 to work?

**Comment by pkerichang** [comment]
Hello,

I hacked the qt_compat.py and found out the error: libGL.so is not found.  For some reason anaconda decided to not install that (probably because of the bash-on-windows weirdness).  However, after running:

sudo apt-get install mesa-utils

matplotlib in anaconda is able to find the openGL library in the system path.  I was able to generate a 3D plot so that worked pretty well.  You might still get some errors like:

QXcbShmImage: shmget() failed (38: Function not implemented) for size 1384960 (640x541)

but that's just bash on windows not supporting some IPC protocol and there's nothing you can do about it.  The plot still shows up successfully though.

**Comment by cvax** [comment]
@pkerichang Thanks. I also saw [this](https://bugreports.qt.io/browse/QTBUG-56419) so looks like it might get a fix in the next version.
